react/hooks/useSession.ts

```diff
@@ -0,0 +1,21 @@
+import { useQuery } from 'react-apollo'
+import { useEffect, useState } from 'react'
+import sessionQuery from 'vtex.store-resources/QuerySession'
+
+const useSession = (): Session | undefined => {
```

Comment:
Instead of this hook, I think you can use [session-client](https://github.com/vtex-apps/session-client)'s useFullSession hook to get the data you want.